src/lib/components/Sidebar/Sidebar.tsx

```diff
@@ -48,14 +50,14 @@ const SidebarComponent: FC<SidebarProps> = ({
 
   return (
     <SidebarContext.Provider value={{ isCollapsed }}>
-      <aside
-        aria-label="Sidebar"
+      <Component
+        aria-label="Navigation Sidebar"
```

Comment:
If this is default `nav`, labelling it with navigation is redundant to screen readers, which will announce the section as a navigation with label navigation sidebar. See redundant descriptions: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#best_practices